### PR TITLE
supplements for provision.sh webserver setup; fixes #914

### PR DESCRIPTION
**Pull request by following5**
This is not tested but just copied from my devel VM setup. If it works,

- http://local.opencaching.de/badrequest.php should show the 404.php page contents
- http://local.opencaching.de/mystatpic.php should show your statistics picture
- http://local.opencaching.de/okapi/update shoud show instructions on installing OKAPI
- http://local.opencaching.de/OC1234 should redirect to http://local.opencaching.de/viewcache.php?cacheid=107691.

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -140,6 +149,9 @@ AddDefaultCharset UTF-8
     MIMEMagicFile conf/magic
 </IfModule>
 
+RewriteEngine On
+RewriteRule ^/((OC|GC)[A-Za-z0-9]{1,5})$ /searchplugin.php?userinput=$1 [NC]
```
Comment (teiling88) [inline provision.sh]: please change $ with \$ and $1 with \$1
after that move it please under ErrorDocument 404 entry.

**Comment by teiling88** [comment]
one little change and it is ready to merge

**Comment by following5** [comment]
I have moved the Rewrite setup up below the 404.
The syntax is correct and identical to the production server setup; your proposed change does not work.

**Comment by teiling88** [comment]
with $ start variables like in php so please escape them otherwise there are missing in httpd.conf

**Comment by following5** [comment]
ok, done

**Pull request merged**